e2e/portalicious/pages/UsersPage.ts

```diff
@@ -69,6 +87,42 @@ class UsersPage extends BasePage {
 
     expect(formattedTextContent).toContain(textContent);
   }
+
+  async addNewUser({ fullName, email }: { fullName: string; email: string }) {
+    // Act
+    await this.newUserButton.click();
+    await this.fullNameInput.fill(fullName);
+    await this.emailInput.fill(email);
+    await this.submitButton.click();
+    // Assert
+    await this.validateToastMessage('User added');
+    await this.validateRowTextContent({
+      email,
+      textContent: `${fullName} ${email}`,
+    });
+  }
+
+  async selectUsersMenuItem({
+    email,
+    menuItem,
+  }: {
+    email: string;
+    menuItem: string;
+  }) {
+    const selectUser = this.page.getByRole('row', {
+      name: email,
+    });
+    await selectUser.getByRole('button').click();
+    await this.page.getByRole('menuitem', { name: menuItem }).click();
+  }
+
+  async resetUsersPassword(email: string) {
+    // Act
+    await this.selectUsersMenuItem({ email, menuItem: 'Reset password' });
+    await this.resetPasswordButton.click();
+    // Assert
+    await this.validateToastMessage('Password reset');
```

Comment:
It is actually a really good point and I also believe they should be split. It is an oversight from my side since validation and action act separately. As well as assertion should be done it test itself. Thank you for catching this.

Reply:
I changed it here and in other places.